packages/squiggle-lang/src/js/cli/index.ts

```diff
@@ -0,0 +1,61 @@
+import fs from "fs";
+import { OutputMode, run } from "./utils";
+
+import { program } from "@commander-js/extra-typings";
+import open from "open";
+
+program
+  .command("run")
+  .arguments("[filename]")
+  .option(
+    "-e, --eval <code>",
+    "run a given squiggle code string instead of a file"
+  )
+  .option("-t --time", "output the time it took to evaluate the code")
+  .option("-q, --quiet", "don't output the results and bindings") // useful for measuring the performance or checking that the code is valid
```

Comment:
Can you add the comment to the string description? Is it important that description is short?

Reply:
I don't think as a user you need to be told when to pass `-q`: if you're getting output that you don't want then you'll look for roughly this description. So the comment more explains why we've provided the option.

Reply:
Description is shown on `--help` in a single line, so shortness is pretty important; as a user, you want read that output quickly.

(Ideally, we should implement a longer man-like page on `--help` and a shorter description on `-h`, like `git` does)